README.md

```diff
@@ -221,6 +221,40 @@ this functionality, set the environment variable `TRUE_LAYER_ENABLE_MOCK` to `"t
 
 The URL for this service should be set using the environment variable `CHECK_FINANCIAL_ELIGIBILITY_HOST`
 
+---
+
+## Geckoboard Dashboard
+
+Several sets of statistics are exported to Geckoboard for displaying on an application dashboard.
+
+### How to add a new widget to the dashboard
+
+There are three steps to creating a new widget for the Geckoboard dashboard 
+
+### 1. Create a widget data provider
+
+Create a new class in the `app/models/dashboard/widget_data_providers` directory. This should define three class methods:
+
+* `.handle` - the name of the widget, which will be qualified with a project name and environment.  For example `my_widget` would become `apply_for_legal_aid.production.my_widget` in the list of datasets on Geckoboard
+* `.dataset_defintion` - the list of fields that will be in the dataset (see https://developer.geckoboard.com/hc/en-us/sections/360002865451-Getting-started for details on how to define and provide data for a dataset.)
+* `.data` - the actual data that will be sent to Geckoboard every time it is run.
+
+This data provider will be used by the `Dashboard::Widget` class when called with the name of the data provider as a parameter.
+
+### 2. Add a cronjob to run it
+
+*** instructions on how to setup a cron job go here ****
```

Comment:
Probably should put the instructions here before merging!

Reply:
Thanks , done.